test/smoke/testApps/HttpDependency/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'war'
+
+dependencies {
+    compile aiCoreJar
+    compile aiWebJar
+    compile 'com.google.guava:guava:20.0'
+
+    providedCompile 'javax.servlet:javax.servlet-api:3.0.1'
+
+    compile 'org.apache.httpcomponents:httpclient:4.5.3'
```

Comment:
We would need to check for other versions of ApacheHttpClient too. May be a seperate test or whatever better way @littleaj knows of. ApacheHttpClient 4.5.6 in particular.

Reply:
Thanks for your remind. I will discuss with @littleaj  about this.